hal/inc/cellular_hal_constants.h

```diff
@@ -39,8 +39,8 @@ typedef void (*_CELLULAR_SMS_CB_MDM)(void* data, int index);
 
 #define DEFINE_NET_PROVIDER_DATA \
     DEFINE_NET_PROVIDER( CELLULAR_NETPROV_TELEFONICA, "spark.telefonica.com", (23*60), (5684) ),  \
-    DEFINE_NET_PROVIDER( CELLULAR_NETPROV_TWILIO, "wireless.twilio.com", (23*60), (4500) ),  \
-    DEFINE_NET_PROVIDER( CELLULAR_NETPROV_KORE, "", (23*60), (5684) ),  \
+    DEFINE_NET_PROVIDER( CELLULAR_NETPROV_KORE_VODAFONE, "vfd1.korem2m.com", (23*60), (5684) ),  \
+    DEFINE_NET_PROVIDER( CELLULAR_NETPROV_KORE_ATT, "", (23*60), (5684) ),  \
```

Comment:
Gen 3 sets the APN for Kore/AT&T to `10569.mcs`, and the user name/password for Kore/Vodafone to `kore`/`kore`. Should we do the same on Gen 2?

Reply:
We have been relying on LTE SIMs to populate the APN, but I will add the APN for Kore AT&T to be explicitly set.  We have determined that the username/password is not necessary on Gen 2 due to how the data link is established currently.